packages/playground-examples/copy/pt/sections.json

```diff
@@ -0,0 +1,47 @@
+{
+  "sections": [
+    {
+      "name": "JavaScript",
+      "id": "JavaScript",
+      "subtitle": "Veja como o TypeScript aprimora o dia a dia trabalhando com JavaScript com uma sintaxe adicional mínima."
+    },
+    {
+      "name": "TypeScript",
+      "id": "TypeScript",
+      "subtitle": "Explore como o TypeScript estende o JavaScript para adicionar mais segurança e ferramentas."
+    },
+    {
+      "name": "3.7",
+      "id": "3.7",
+      "subtitle": "Consulte <a href='https://devblogs.microsoft.com/typescript/announcing-typescript-3-7/'>Notas da versão</a>."
+    },
+    {
+      "name": "Playground",
+      "id": "Playground",
+      "subtitle": "Saiba o que mudou neste site."
+    }
+  ],
+
+  "sortedSubSections": [
+    // JS
+    "JavaScript Essentials",
+    "Functions with JavaScript",
+    "Working With Classes",
+    "Modern JavaScript",
+    "External APIs",
+    "Helping with JavaScript",
+    // TS
+    "Primitives",
+    "Type Primitives",
+    "Meta-Types",
+    "Language",
+    "Language Extensions",
+    // Examples
+    "Syntax and Messaging",
+    "Types and Code Flow",
+    "Fixits",
+    // Playground
+    "Config",
+    "Tooling"
```

Comment:
This uses filenames, so it might be redundant. In the future I might remove this from non-english to simplify translator's jobs.